extensions/sharing/pkg/revaconfig/config.go

```diff
@@ -73,17 +73,6 @@ func SharingConfigFromStruct(cfg *config.Config) map[string]interface{} {
 							"enable_expired_shares_cleanup": cfg.PublicSharingDrivers.SQL.EnableExpiredSharesCleanup,
 							"janitor_run_interval":          cfg.PublicSharingDrivers.SQL.JanitorRunInterval,
 						},
-						"owncloudsql": map[string]interface{}{
```

Comment:
why remove the PublicSharingDrivers?

Reply:
because there is no owncloudsql publicshare driver in reva: https://github.com/cs3org/reva/blob/afc8ead0c9a6a0acc5b46bf27a8b18ad458c098d/pkg/publicshare/manager/loader/loader.go#L23-L25

```
package loader

import (
	// Load core share manager drivers.
	_ "github.com/cs3org/reva/v2/pkg/publicshare/manager/cs3"
	_ "github.com/cs3org/reva/v2/pkg/publicshare/manager/json"
	_ "github.com/cs3org/reva/v2/pkg/publicshare/manager/memory"
	// Add your own here
)
```